app/src/androidTest/java/com/hedvig/app/feature/loggedin/NavBarTest.kt

```diff
@@ -0,0 +1,49 @@
+package com.hedvig.app.feature.loggedin
+
+
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.rule.ActivityTestRule
+import com.agoda.kakao.screen.Screen.Companion.onScreen
+import com.hedvig.android.owldroid.graphql.LoggedInQuery
+import com.hedvig.app.feature.loggedin.ui.LoggedInActivity
+import com.hedvig.app.testdata.feature.referrals.LOGGED_IN_DATA_WITH_REFERRALS_FEATURE_ENABLED
+import com.hedvig.app.util.ApolloCacheClearRule
+import com.hedvig.app.util.ApolloMockServerRule
+import com.hedvig.app.util.apolloResponse
+import com.hedvig.app.util.hasNumberOfMenuItems
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+
+@RunWith(AndroidJUnit4::class)
+class NavBarTest {
+    @get:Rule
+    val activityRule = ActivityTestRule(LoggedInActivity::class.java, false, false)
+
+    @get:Rule
+    val mockServerRule = ApolloMockServerRule(
+        LoggedInQuery.QUERY_DOCUMENT to apolloResponse {
+            success(
+                LOGGED_IN_DATA_WITH_REFERRALS_FEATURE_ENABLED
+            )
+        }
+    )
+
+    @get:Rule
+    val apolloCacheClearRule = ApolloCacheClearRule()
+
+    @Test
+    fun shouldShowCorrectIconOnNavBar() {
```

Comment:
This test is named to assume that this is always the amount of `BottomNavigation`-items, which I believe is incorrect (there are currently at least 2 variations possible)